apps/aeutils/src/aeu_db.erl

```diff
@@ -0,0 +1,141 @@
+%%%=============================================================================
+%%% @copyright (C) 2019, Aeternity Anstalt
+%%% @doc
+%%%    Helpers to change node name in Mnesia
+%%% @end
+%%%=============================================================================
+-module(aeu_db).
+
+%% API
+-export([change_node/3]).
+
+-define(db_renaming_error(E), {db_renaming_error, E}).
+
+%%%===================================================================
+%%% API
+%%%===================================================================
+
+change_node(SchemaDATFilePath, FromNode, ToNode) when is_list(SchemaDATFilePath)
+                                                      andalso is_atom(FromNode)
+                                                      andalso is_atom(ToNode) ->
+    try do_change_node(SchemaDATFilePath, FromNode, ToNode)
+    catch throw:?db_renaming_error(Error) -> {error, Error}
+    end.
+
+%%%===================================================================
+%%% Internal functions
+%%%===================================================================
+
+do_change_node(SchemaDatFile, FromNode, ToNode) ->
+    ok      = assert_file_present(SchemaDatFile),
+    ok      = prepare_schema_dat_backup(SchemaDatFile),
+    TabName = dets_open_file(SchemaDatFile),
+    ok      = dets_change_node(FromNode, ToNode, TabName),
+    ok      = dets_sync(TabName),
+    ok      = dets_close(TabName).
+
+assert_file_present(SchemaDatFile) ->
+    case filelib:is_regular(SchemaDatFile) of
+        true -> ok;
+        false ->
+            io:fwrite("Schema file not found: ~p~n", [SchemaDatFile]),
+            db_renaming_error(file_not_found)
+    end.
+
+prepare_schema_dat_backup(SchemaDatFile) ->
+    BackupFile = get_backup_file(SchemaDatFile),
+    case file:copy(SchemaDatFile, BackupFile) of
```

Comment:
```suggestion
    case file:copy(SchemaDatFile, {BackupFile, [sync]}) of
```
?

Reply:
I see this ok in OTP 20.1 too http://erlang.org/documentation/doc-9.1/lib/kernel-5.4/doc/html/file.html#copy-2